nettacker/modules/vuln/n8n_cve_2026_21858.yaml

```
@@ -0,0 +1,49 @@
+info:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)